pep8.py

```diff
@@ -2061,7 +2061,8 @@ def read_config(options, args, arglist, parser):
         # Second, parse the configuration
         for opt in config.options(pep8_section):
             if opt.replace('_', '-') not in parser.config_options:
-                print("  unknown option '%s' ignored" % opt)
+                if options.verbose:
+                    print("  unknown option '%s' ignored" % opt)
```

Comment:
@ABaldwinHunter Are you sure `--quiet` accomplishes negating the verbose configuration?